docs/HACKING.md

```diff
@@ -7,7 +7,8 @@
 - The operator is written using operator-sdk 
 [v1.6.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.6.1) and has the same 
 [pre-requisites](https://sdk.operatorframework.io/docs/installation/#prerequisites) as it does.
-
```

Comment:
Avoid removing white space if not necessary, removing the whitespace here removes the separation between the pre-reqs and the next section.